src/Helpers/Task/Steps/InitNextjsApp.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace AcquiaCMS\Cli\Helpers\Task\Steps;
+
+use AcquiaCMS\Cli\Helpers\Process\Commands\Drush;
+
+/**
+ * Run the drush command to import site studio packagess.
+ */
+class InitNextjsApp {
+
+  /**
+   * A drush command object.
+   *
+   * @var \AcquiaCMS\Cli\Helpers\Process\Commands\Drush
+   */
+  protected $drushCommand;
+
+  /**
+   * Constructs an object.
+   *
+   * @param \AcquiaCMS\Cli\Helpers\Process\Commands\Drush $drush
+   *   Holds the drush command class object.
+   */
+  public function __construct(Drush $drush) {
```

Comment:
@rajeshreeputra Let's now use DI, instead of directly accessing the class.
```suggestion
  public function __construct(ContainerInterface $container) {
    $this->drushCommand = $container->get(Drush::class);
  }
```

Reply:
I thought the point of services and DI was so classes were not dependant on the service container?

Reply:
@fiasco Ahh correct. But I think that's how currently the symfony auto-wiring the service dependencies in our case i.e using the complete class namespace whereas it should be based on some unique name, similar to how Drupal allows to create/manage service objects. I've that in mind to look & improve this, across all places in our project. :)

Reply:
updated